spec/models/user_spec.rb

```diff
@@ -12,4 +12,8 @@
     it {is_expected.to have_db_column :email}
     it {is_expected.to have_db_column :encrypted_password}
   end
+
+  context `validations` do
+    it {is_expected.to have_one(:cart)}
```

Comment:
Why just one?

Reply:
We are sticking with this solution, it's simpler solution